repos/config/detail/ros2_config.bzl

```
@@ -30,13 +30,41 @@ _archive_attrs = {
     ),
 }
 
+BUILD_FILE_CONTENT = """\
+load("@rules_ros//repos/config/detail:generate_repos_lock.bzl", "generate_repos_lock")
+
+generate_repos_lock(
+    name = "repos_lock.update",
+    repos_file = "ros.repos",  # Custom repos file
+    setup_file = "setup.bzl",  # Custom setup file
```

Those comment don't serve any purpose
```suggestion
    repos_file = "ros.repos",
    setup_file = "setup.bzl",
```